examples/howto_use_legacy_code/README.md

```diff
@@ -14,7 +14,11 @@ any external tool to compile some code. Often this function also downloads
 the code from some external repository, but it can use code in your source
 tree.
 
-[ExternalProject_Add]: https://cmake.org/cmake/help/latest/module/ExternalProject.html
+> **WARNING:** the development version of this document may not work with the
```

Comment:
`s/the/The/` (et seq)

The usual rule is to capitalize after a colon if it starts a complete sentence.

Reply:
Done.